garrysmod/lua/autorun/game_hl2.lua

```diff
@@ -19,22 +19,30 @@ Category = "Half-Life 2"
 
 -- Ammo
 ADD_ITEM( "AR2 Ammo",			"item_ammo_ar2" )
+ADD_ITEM( "Large AR2 Ammo",		"item_ammo_ar2_large" )
 ADD_ITEM( "Pistol Ammo",		"item_ammo_pistol" )
+ADD_ITEM( "Large Pistol Ammo",	"item_ammo_pistol_large" )
 ADD_ITEM( "Shotgun Ammo",		"item_box_buckshot" )
 ADD_ITEM( "357 Ammo",			"item_ammo_357" )
+ADD_ITEM( "Large 357 Ammo",		"item_ammo_357_large" )
 ADD_ITEM( "SMG Ammo",			"item_ammo_smg1" )
-ADD_ITEM( "Combine Balls",		"item_ammo_ar2_altfire" )
+ADD_ITEM( "Large SMG Ammo",		"item_ammo_smg1_large" )
+ADD_ITEM( "AR2 Orb",			"item_ammo_ar2_altfire" )
 ADD_ITEM( "Crossbow Bolts",		"item_ammo_crossbow" )
 ADD_ITEM( "SMG Grenade",		"item_ammo_smg1_grenade" )
-ADD_ITEM( "RPG",				"item_rpg_round" )
+ADD_ITEM( "RPG Rocket",			"item_rpg_round" )

```

Comment:
Perhaps spaces should be used for alignment here instead of tabs; as you can see in the GitHub editor, some (most?) editors fail when tabs are used for alignment.